v2/guidelines.html

```diff
@@ -70,5 +70,25 @@
 <section id="abstract">
     <p></p>
 </section>
+<section class="informative">
+    <h2>General Principles</h2>
+    <div class="issue" data-number="30"></div>
+    <p>
+        Music incipits help identify works and facilitate the comparison of historical musical sources.
+    </p>
```

Comment:
I think we would still need to have an introduction paragraph at the beginning of the specs

Reply:
Yes, I agree -- but we can cover this in a different issue, #57